files/com.inform7.IDE.appdata.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  ​<id>com.inform7.IDE</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-Compiler-InformLicense-EverthingElseGPL-3.0</project_license>
+  <content_rating type="oars-1.0" />
+  <name>Inform 7</name>
+  <summary>Write interactive fiction</summary>
+  <description>
+    <p>
+      Inform is a design system for interactive fiction based on natural
+      language.
+    </p>
+    <p>
+      Interactive fiction is a literary form which involves programming a
+      computer so that it presents a reader with a text which can be explored.
+      Inform aims to make the burden of learning to program such texts as light
+      as possible.
+    </p>
+  ​</description>
+  <launchable type="desktop-id">com.inform7.IDE.desktop</launchable>
+  <url type="homepage">http://inform7.com/</url>
+  <provides><binary>gnome-inform7</binary></provides>
```

Comment:
Not needed; It's extracted from the desktop file.

Reply:
Removed.